app/javascript/components/SignUp.jsx

```diff
@@ -1,116 +1,86 @@
 import React from 'react';
-import Axios from 'axios';
+import SignUpAttendee from './SignUpAttendee';
+import SignUpOrganizer from './SignUpOrganizer';
+import { makeStyles } from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import PropTypes from 'prop-types';
+import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
 
-export default class SignUpForm extends React.Component {
+export default class SignUp extends React.Component {
```

Comment:
Why it was changed? I think SignUpForm is better name for component than just SignUp